db/migrate/20240729103525_update_proxy_byos_column_type.rb

```diff
@@ -1,11 +1,11 @@
 class UpdateProxyByosColumnType < ActiveRecord::Migration[6.1]
   def up
-    add_column :systems, :proxy_byos_mode, :integer, default: 0
-    System.where(proxy_byos: false).in_batches.update_all proxy_byos_mode: 0
+    add_column :systems, :proxy_byos_mode, :integer, default: nil
     System.where(proxy_byos: true).in_batches.update_all proxy_byos_mode: 1
   end
 
   def down
+    System.where(proxy_byos_mode: :byos).in_batches.update_all proxy_byos: true
```

Comment:
My understanding is that those system would get back being `payg` instead of `hybrid` which in the `proxy_byos` context means `false` or `0`

Reply:
I don't think the case where `proxy_byos_mode: hybrid` is relevant. It would imply that we managed to update the server in the update infrastructure to RMT 2.19 or greater, see my other comments and then decide to to backwards.

Reply:
which comment, @rjschwei ?

I guess we can remove line 8 as the `proxy_byos` column isn't getting removed